lib/tasks/db.rake

```diff
@@ -0,0 +1,46 @@
+namespace :db do
+
+  desc 'Dumps the database to file'
+  task :dump => :environment do
+
+    host = ActiveRecord::Base.connection_config[:host]
+    db = ActiveRecord::Base.connection_config[:database]
+
+    cmd = "pg_dump --no-owner --no-acl -v -c -f #{dump_file} --no-password "
+    cmd += ENV['DATABASE_URL'] || "--host #{host} --dbname #{db}"
```

Comment:
Thinking about it more, it's not really just for testing, maybe you would want to use the rake task to dump the database out if you had a local db...
The line in question is just about adapting `database.yml` and the current environment to pg_dump. I can re-write it so it's a bit clearer...